### PR TITLE
include warning message in tidb control

### PR DESCRIPTION
**Pull request by songrijie**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Include warning messages to notify customers. We don't recommend to use tidb-ctl in application design.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.2 (TiDB 7.2 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v7.0 (TiDB 7.0 versions)
- [x] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/14041
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-control.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ aliases: ['/docs-cn/dev/tidb-control/','/docs-cn/dev/reference/tools/tidb-contro
 
 TiDB Control 是 TiDB 的命令行工具，用于获取 TiDB 状态信息，多用于调试。本文介绍了 TiDB Control 的主要功能和各个功能的使用方法。
 
+> **注意：**
+>
+> TiDB Control 主要用作诊断调试， 不保证和 TiDB 未来引入的新特性完全兼容。 因此不推荐客户在应用程序开发或工具开发中利用 TiDB Control 获取结果。
```
Comment (lilin90) [inline tidb-control.md:12]:
```suggestion
> TiDB Control 主要用诊断调试，不保证和 TiDB 未来引入的新特性完全兼容。因此不推荐客户在应用程序开发或工具开发中利用 TiDB Control 获取结果。 
```

Comment (songrijie) [reply tidb-control.md:12]: LGTM

**Review by tiancaiamao — APPROVED** [review]

**Review by lilin90 — COMMENTED** [review]
/lgtm

**Pull request merged**